src/main/java/org/reactome/release/qa/graph/TwoAttributesReferToSameCheck.java

```diff
@@ -90,17 +94,19 @@ protected void executeQACheck(CheckConfiguration config, QAReport report) throws
             if (isEscaped(inst)) {
                 continue;
             }
-            Long valueId = result.getLong(2);
-            GKInstance value = dba.fetchInstance(valueId);
-            if (value == null)
-                throw new IllegalStateException("Instance cannot be found for " + valueId + ".");
-            report.addLine(inst.getDBID() + "",
-                           inst.getDisplayName(),
-                           inst.getSchemClass().getName(),
-                           config.toString(),
-                           value.getDBID() + "",
-                           value.getDisplayName(),
-                           QACheckerHelper.getLastModificationAuthor(inst));
+            if (!skipList.containsInstanceDbId(inst)) {
+                Long valueId = result.getLong(2);
```

Comment:
It looks like this is pre-existing code, but it might be better to use a primitive “long” rather than an object “Long” for the variable type.

Reply:
I've updated this value and the one above it (dbId)